### PR TITLE
feat(carousel): +carousel component for ML feature

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
### Proposed Changes

Added the carousel package to plasma-mantine package for ML feature 

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/mantine/package.json [hunk 1]
```diff
@@ -30,11 +30,13 @@
     "dependencies": {
         "@coveord/plasma-react-icons": "workspace:*",
         "@coveord/plasma-tokens": "workspace:*",
+        "@mantine/carousel": "^5.6.4",
```
Comment (aiheon) [inline packages/mantine/package.json:33]:
```suggestion
        "@mantine/carousel": "5.6.4",
```

File: packages/mantine/package.json [hunk 2]
```diff
@@ -30,11 +30,13 @@
...
         "@monaco-editor/react": "4.4.5",
         "@swc/helpers": "0.4.11",
         "@tanstack/react-table": "8.5.11",
         "@tanstack/table-core": "8.5.11",
         "dayjs": "1.11.3",
+        "embla-carousel-react": "^7.0.4",
```
Comment (aiheon) [inline packages/mantine/package.json:39]:
```suggestion
        "embla-carousel-react": "7.0.4",
```

**Review by aiheon — APPROVED** [review]

File: packages/mantine/package.json [hunk 3]
```diff
@@ -30,11 +30,13 @@
     "dependencies": {
         "@coveord/plasma-react-icons": "workspace:*",
         "@coveord/plasma-tokens": "workspace:*",
+        "@mantine/carousel": "5.6.4",
```
Comment (GermainBergeron) [inline packages/mantine/package.json:33]: We put the mantine stuff in the devDependencies and peerDependencies to allow us to have a different version in packages that install plasma-mantine

File: packages/mantine/package.json [hunk 4]
```diff
@@ -30,11 +30,13 @@
...
         "@monaco-editor/react": "4.4.5",
         "@swc/helpers": "0.4.11",
         "@tanstack/react-table": "8.5.11",
         "@tanstack/table-core": "8.5.11",
         "dayjs": "1.11.3",
+        "embla-carousel-react": "7.0.4",
```
Comment (GermainBergeron) [inline packages/mantine/package.json:39]: Same thing for that, it should be a dev + peer dependency

**Pull request merged**